doc/manual.wiki

```diff
@@ -345,7 +345,7 @@ let log msg =
   in
   Lwt_io.printlf "%s: %s" thread_id msg
 
-lwt () =
+let lwt =
```

Comment:
I think this should be `let%lwt () = ...`

Reply:
Cool, thanks for helping me learn the right way to do it along the way. :)